addons/auth_signup/data/auth_signup_data.xml

```diff
@@ -3,6 +3,8 @@
     <data noupdate="1">
         <!-- activate B2C portal by default -->
         <function model="ir.config_parameter" name="set_param" eval="('auth_signup.invitation_scope', 'b2c')"/>
+        <!-- activate Reset password by default -->
+        <function model="ir.config_parameter" name="set_param" eval="('auth_signup.reset_password', 'True')"/>
```

Comment:
why don't you just use `default=True` in https://github.com/odoo/odoo/blob/master/addons/auth_signup/models/res_config_settings.py#L12 ?

Reply:
Hello @mge-odoo 
As config_parameter was set on the field, I thought why not to use it.  
And if we use default=True in field definition, it is throwing error while opening general setting menu due to, as value is True https://github.com/odoo/odoo/blob/master/odoo/addons/base/models/res_config.py#L545

Let me know your thoughts on it.

Reply:
You are right, as `ir.config_parameter` values are stored as `Text`, the default value should be 'True' instead ; but I don't know if this works. Let me now if it does ; if it doesn't, then we'll go with your solution :) Thanks mate.

Reply:
Hello @mge-odoo , 
I've checked with True and 'True', In both the cases it is not working. 
Actually, I started the development with that solution only then I switched to current solution. 

Thanks,